tests/returntypes/transforms/test_batch_params_new.py

```diff
@@ -487,7 +487,7 @@ def test_jax(self, diff_method, tol):
 
         dev = qml.device("default.qubit", wires=2)
 
-        @qml.batch_params
+        @qml.batch_params(all_operations=True)

```

Comment:
Just out of curiosity: How does this change come about? :D 
Is it something users have to change as well?

Reply:
Only trainable parameters are considered for this transform by default, previously if all parameters are jax Arrays we would consider trainable, but it is not the case anymore. As batch_input and batch_params are covered by broadcasting, I don't think it is worth adding argnum for Jax.